addons/stock/models/stock_move_line.py

```diff
@@ -142,12 +142,11 @@ def _compute_quantity(self):
         for record in self:
             if not record.quant_id or record.quantity:
                 continue
-            if float_compare(record.move_id.product_qty, record.quantity, precision_rounding=record.move_id.product_uom.rounding) > 0:
+            if record.move_id and float_compare(record.move_id.product_qty, record.move_id.quantity, precision_rounding=record.move_id.product_uom.rounding) > 0:
```

Comment:
```suggestion
            if record.move_id and float_compare(record.move_id.product_qty, record.quantity_product_uom, precision_rounding=record.move_id.product_uom.rounding) > 0:
```
We want to compare quantities in the same uom.
- `move_id.product_qty` is in the ***product***'s uom.
- `record.quantity` is in the ***move line***'s uom.
- `record.quantity_product_uom` is in the ***product***'s uom.